plugins/BEdita/Core/src/Model/Entity/Stream.php

```diff
@@ -248,4 +254,16 @@ protected function _getUrl()
 
         return $this->_properties['url'] = FilesystemRegistry::getPublicUrl($this->uri);
     }
+
+    protected function _getMetadata($source)
+    {
+        $ext_allowed = Configure::read('Streams.ext_allowed');
+        if (in_array($this->mime_type, $ext_allowed)) {
+            try {
+                $this->file_metadata = json_encode(exif_read_data($source));
```

Comment:
remove `json_encode` and cast to `(array)` => this way we will have a json object, not a string...